build.gradle.kts

```diff
@@ -66,7 +66,7 @@ dependencies {
    Uncomment for testing with Intellij IDEA
```

Comment:
Please, add a comment which warns to change the test mode before building in the file